aws/table_aws_mgn_application.go

```diff
@@ -0,0 +1,202 @@
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"UninitializedAccountException"}),
+			},
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "application_id", Require: plugin.Optional},
+				{Name: "wave_id", Require: plugin.Optional},
+				{Name: "is_archived", Require: plugin.Optional, Operators: []string{"=", "<>"}},
+			},
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "Application name.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "application_id",
+				Description: "Application ID.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ApplicationID"),
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the application",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "creation_date_time",
+				Description: "Application creation dateTime.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "description",
+				Description: "Application description.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "is_archived",
+				Description: "Application archival status.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "last_modified_date_time",
+				Description: "Application last modified dateTime.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "wave_id",
+				Description: "Application wave ID.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("WaveID"),
+			},
+			{
+				Name:        "application_aggregated_status",
+				Description: "Application aggregated status.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "tags",
+				Description: "A list of tags attached to the Cluster.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Arn").Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+// // LIST FUNCTION
+func ListAwsMGNApplication(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	svc, err := MGNClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_mgn_application.ListAwsMGNApplication", "connection_error", err)
+		return nil, err
+	}
+	if svc == nil {
+		// Unsupport region, return no data
+		return nil, nil
+	}
+
+	// Reduce the basic request limit down if the user has only requested a small number of rows
+	maxLimit := int32(100)
```

Comment:
```suggestion
	maxLimit := int32(1000)
```
As per the documentation, I think the max limit should be `1000` - https://docs.aws.amazon.com/mgn/latest/APIReference/API_ListApplications.html